Resources/Prototypes/SS220/Language/language.yml

```
@@ -0,0 +1,176 @@
+# Based on SS13 Paradise (https://github.com/ParadiseSS13/Paradise/blob/017a56f8080361f04c296c94859802aaddf25114/code/modules/mob/language.dm#L4)
+
+# Everyone understands you, you understand everyone, but there's no right to speak all languages
+- type: languages
+  id: Universal
+  syllables: []
+
+# Galactic common lanaguges region:
+- type: languages
+  id: Galactic
+  name: "language-galacticcommon-name"
+  description: "language-galacticcommon-desc"
+  spaceChance: 50
+  syllables: [бла, бла, бла, бле, мех, ньех, нах, ах, угу, увы]
```

```suggestion
  syllables: [бла, бла, бла, бле, мех, ньех, нах, ах, угу]
```
Убери брейнрот пожалуйста (да, фраза `"увы"` - брейнрот всея 220)

---

> Убери брейнрот пожалуйста (да, фраза `"увы"` - брейнрот всея 220)

это и есть отсылка на брейрот

---

О НЕТ